javascript/ui/element/emailtesthelper.js

```diff
@@ -105,7 +105,7 @@ element.EmailTestHelper.prototype.testOnTextChangedClearError_ = function() {
   this.checkErrorShown(error,
       firebaseui.auth.soy2.strings.errorMissingEmail().toString());
 
-  // Emulate that a 'u' is typed in to name input.
+  // Emulate that a 'u' is typed into name input.
```

Comment:
`in to`/`into` tend to be controversial. I believe `typed into <container>` is the appropriate preposition form, as opposed to `typed in <to name [the] input>`.